zeppelin-web/src/app/notebook/paragraph/paragraph.css

```diff
@@ -170,7 +170,7 @@
   position: absolute;
   clear: both;
   right: 15px;
-  top: 16px;
+  top: 21px;
```

Comment:
It should actually be 20px (Latest PR addresses it). The reason I changed it is because with the new PR, the controls in paragraphs moved and it need to be changed as well. See below: 

Before the PR: 
![before pr](https://cloud.githubusercontent.com/assets/6380209/11505825/ed34774e-9811-11e5-8125-cb6750787e5e.png)

If we left it at 16px:
![16px](https://cloud.githubusercontent.com/assets/6380209/11505830/f19ee166-9811-11e5-8645-f8c2721c9394.png)

With 20px:
![20px](https://cloud.githubusercontent.com/assets/6380209/11505832/f1c55620-9811-11e5-8802-cecabd84abf0.png)

Let me know if I'm missing something.